cvat/schema.yml

```diff
@@ -12,6 +12,100 @@ info:
     name: MIT License
     url: https://en.wikipedia.org/wiki/MIT_License
 paths:
+  /api/analytics/reports:
+    get:
+      operationId: analytics_get_reports
+      description: Receive analytics report
+      summary: Method returns analytics report
+      parameters:
+      - in: query
+        name: end_date
+        schema:
+          type: string
+          format: date-time
+        description: Specify the end date for filtering report data.
+      - in: query
+        name: job_id
+        schema:
+          type: integer
+        description: Specify job ID
+      - in: query
+        name: project_id
+        schema:
+          type: integer
+        description: Specify project ID
+      - in: query
+        name: start_date
+        schema:
+          type: string
+          format: date-time
+        description: Specify a start date for filtering report data.
+      - in: query
+        name: task_id
+        schema:
+          type: integer
+        description: Specify task ID
+      tags:
+      - analytics
+      security:
+      - sessionAuth: []
+        csrfAuth: []
+        tokenAuth: []
+      - signatureAuth: []
+      - basicAuth: []
+      responses:
+        '200':
+          content:
+            application/vnd.cvat+json:
+              schema:
+                $ref: '#/components/schemas/AnalyticsReport'
+          description: ''
+        '404':
+          description: Not found
+    post:
+      operationId: analytics_create_report
+      summary: Creates a analytics report asynchronously and allows to check request
+        status
+      parameters:
+      - in: query
+        name: rq_id
+        schema:
+          type: string
+        description: |
+          The report creation request id. Can be specified to check the report
+          creation status.
+      tags:
+      - analytics
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AnalyticsReportCreateRequest'
+      security:
+      - sessionAuth: []
+        csrfAuth: []
+        tokenAuth: []
+      - signatureAuth: []
+      - basicAuth: []
+      responses:
+        '201':
+          content:
+            application/vnd.cvat+json:
+              schema:
+                $ref: '#/components/schemas/RqId'
+          description: A analytics report request has been computed
```

Comment:
If this is the response when the report has been created, shouldn't it have an empty body (or a body with some information about the report), rather than an `RqId`?